containers/init/ready/ready_test.go

```diff
@@ -245,3 +244,20 @@ func (plm *PodListerMock) List(opts metav1.ListOptions) (*corev1.PodList, error)
 
 	return plm.PodList, nil
 }
+
+type LoadtestGetterMock struct {
+	Loadtest      *grpcv1.LoadTest
+	SleepDuration time.Duration
+	Error         error
+	invocation    int
+}
+
+func (lgm *LoadtestGetterMock) Get(testName string, opts metav1.GetOptions) (*grpcv1.LoadTest, error) {
+	time.Sleep(lgm.SleepDuration)
+
+	if lgm.Error != nil {
+		return nil, lgm.Error
+	}
+
+	return lgm.Loadtest, nil
+}
```

Comment:
Great job! I would only change the name to `LoadTestGetterMock` to match the interface. You are doing a great job with our tests 😄

Reply:
Done, thanks:)